src/algorithms/tonal/audio2midi.cpp

```diff
@@ -0,0 +1,143 @@
+#include "audio2midi.h"
+
+using namespace std;
+using namespace essentia;
+using namespace standard;
+
+const char *Audio2Midi::name = "Audio2Midi";
+const char *Audio2Midi::category = "Pitch";
+const char *Audio2Midi::description = DOC("Wrapper around Audio2Pitch and Pitch2Midi for real time application");
```

Comment:
The documentation does not clarify that the algorithm has a state that is used to estimate note on/off events based on consequent compute() call. We could explicitly mention this. Also related to DOC in Pitch2Midi.